scripts/install/install_deps.bash

```diff
@@ -5,10 +5,20 @@ sudo apt-get clean > /dev/null
 sudo apt-get update > /dev/null
 
 install_dependency() {
-    echo "Installing $1 ..."
-    sudo bash $2 &>install.log
+    echo -n "Installing $1"
 
-    if [[ $? -ne 0 ]]; then
+    if [[ $- == *i* ]]; then # If shell is interactive
+    # Turn off buffering in awk (so dots print right away)
+        local awk_arg="-W interactive"
+    fi
+
+    # Run the install script. Print a dot for every 10 lines of output and save
+    # the full output to a log file
+    sudo bash $2 |& tee install.log \
+        |& awk ${awk_arg} 'NR%10==1 { printf "."} END{ print ""}'
```

Comment:
Just a comment, but this is very clever indeed, my guess is that you're trying to print dots so that we don't need the `travis_wait` in the `travis.yaml` and kill off our build because it didn't print anything for 10 minutes.